src/detectors/FHCAL/CalorimeterHit_factory_LFHCALRecHits.h

```diff
@@ -30,7 +34,7 @@ class CalorimeterHit_factory_LFHCALRecHits : public eicrecon::JFactoryPodioT<edm
 
         // digitization settings, must be consistent with digi class
         m_capADC=65536;//2^16
-        m_dyRangeADC=1 * dd4hep::GeV;
+        m_dyRangeADC=1 * unit::GeV;
         m_pedMeanADC=20;
         m_pedSigmaADC=0.8;
         m_resolutionTDC=10 * dd4hep::picosecond;
```

Comment:
```suggestion
        m_resolutionTDC=10 * unit::ps;
```

Reply:
and many others, I only glanced through and caught a few of these, and `grep` can find the rest

Reply:
I have not touched times and lengths (at least not in the public PR). The `dd4hep::picosecond` is likely a blatant bug here, but we would need to check those cases more carefully.

Reply:
No bug was there. Now updated to use `units::`.